_docs/kustomize/akash-provider/kustomization.yaml

```diff
@@ -19,7 +19,7 @@ configMapGenerator:
     literals:
       - home=/home
       - from=main
-      - node=tcp://akashd:26657
+      - node=http://akashd:26657
```

Comment:
no longer `tcp://`?

Reply:
New sdk changes... seems like a weird config artifact that might get flipped back.